exonum_launcher/supervisor.py

```diff
@@ -105,4 +113,4 @@ def send_deploy_request(self, deploy_request: bytes) -> List[str]:
 
     def send_start_instance_request(self, start_request: bytes) -> List[str]:
```

Comment:
I believe it's now `send_propose_config_request` (docstring should be updated as well).

Reply:
Agree